app/celery/research_mode_tasks.py

```diff
@@ -61,6 +62,69 @@ def send_email_response(reference, to):
     process_ses_results.apply_async([body], queue=QueueNames.RESEARCH_MODE)
 
 
+def send_letter_response(notification_id: uuid.UUID, billable_units: int, postage: str):
+    signed_notification_id = signing.encode(str(notification_id))
+    api_call = (
+        f"{current_app.config['API_HOST_NAME_INTERNAL']}/notifications/letter/status?token={signed_notification_id}"
+    )
+
+    headers = {"Content-type": "application/json"}
+    data = _create_fake_letter_callback_data(notification_id, billable_units, postage)
+
+    try:
+        response = requests_session.request("POST", api_call, headers=headers, data=json.dumps(data), timeout=30)
+        response.raise_for_status()
+    except requests.HTTPError as e:
+        current_app.logger.error("API POST request on %s failed with status %s", api_call, e.response.status_code)
+        raise e
```

Comment:
should we be re-raising everything? or just 5xx.

or perhaps rather, should we handle 5xx differently to 4xx in terms of retries etc